wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/tool/dockerimage/DockerimageToolToWorkspaceApplier.java

```diff
@@ -90,25 +91,9 @@ public void apply(
     String machineName = dockerimageTool.getName();
     MachineConfigImpl machineConfig = new MachineConfigImpl();
 
-    dockerimageTool.getEnv().forEach(e -> machineConfig.getEnv().put(e.getName(), e.getValue()));
```

Comment:
Because now generated deployment contain needed environment variables.
Here I chose to put everything to Deployment except `volumes` and `endpoints`, because the logic is not simple and Che Server already does it.

Reply:
thanks for the explanation